tools/ios/assemble_pod_package.py

```diff
@@ -83,14 +83,19 @@ def assemble_pod_package(
     pod_description = "Pod containing pre and post processing custom ops for onnxruntime."
     with open(framework_info_file, mode="r") as f:
         framework_info = json.load(f)
-        ios_deployment_target = framework_info["apple_deployment_target"]
+        ios_deployment_target = framework_info["iphonesimulator"]["apple_deployment_target"]
+        # NOTE: hard coded the cmake osx sysroot variable for macosx key here. 
```

Comment:
as noted, not sure why sysroot is not setting as `macosx` here.

Reply:
this will need to be updated - the path is too specific

Reply:
yeah I agree. not sure why the cmake_osx_sysroot is not equal to macosx here. will figure out this.